package-lock.json

```diff
@@ -1160,6 +1160,38 @@
       "integrity": "sha512-shAmDyaQC4H92APFoIaVDHCx5bStIocgvbwQyxPRrbUY20V1EYTbSDchWbuwlMG3V17cprZhA6+78JfB+3DTPw==",
       "dev": true
     },
+    "@samverschueren/stream-to-observable": {
+      "version": "0.3.0",
+      "resolved": "http://artifactory.inf.las1.saucelabs.net:80/artifactory/api/npm/all-npm/@samverschueren/stream-to-observable/-/stream-to-observable-0.3.0.tgz",
```

Comment:
@seanenright we should not use `artifactory` as npm registry for network-viewer.